ibm/service/kubernetes/resource_ibm_container_vpc_cluster.go

```diff
@@ -598,7 +601,7 @@ func resourceIBMContainerVpcClusterCreate(d *schema.ResourceData, meta interface
 		workerpool.Labels = labels
 	}
 
-	disableOutboundTrafficProtection := d.Get("disable_outbound_traffic_protection").(bool)
+	disableOutboundTrafficProtection := d.Get(DisableOutboundTrafficProtectionFlag).(bool)
```

Comment:
Unfortunately not, because this parameter is not a fetchable one. It can be given at cluster create time, or can be updated after with a seperate API call. If I'm right all the parameters in the  `resource_ibm_container_vpc_cluster.go` are fetchable ones.